core/src/main/scala/org/dbpedia/extraction/ontology/RdfNamespace.scala

```diff
@@ -65,6 +65,14 @@ object RdfNamespace {
   val MAPPINGS = ns("mappings", "http://mappings.dbpedia.org/wiki/", false)
   val D0 = ns("d0", "http://www.ontologydesignpatterns.org/ont/d0.owl#", false)
   val DUL = ns("dul", "http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#", false)
+  val RR = ns("rr", "http://www.w3.org/1999/02/22-rdf-syntax-ns#")
+  val RML = ns("rml", "http://semweb.mmlab.be/ns/rml#")
+  val QL = ns("ql", "http://semweb.mmlab.be/ns/ql#")
+  val DBO = ns("dbo", "http://dbpedia.org/ontology/")
+  val FNML = ns("fnml", "http://semweb.mmlab.be/ns/fnml#")
+  val FNO = ns("fno", "http://w3id.org/function/ontology/")
+  val EX = ns("ex", "http://example.org/stuff/1.0/")
```

Comment:
I meant this do we need this file now? looks like all references to it are deleted